portfolio/style.css

```diff
@@ -0,0 +1,148 @@
+body {
+  font-family: 'Syncopate', sans-serif;
+}
+
+.container-fluid {
```

Comment:
If you remove your top-level `container` div as I suggested, you can remove these overrides.